tests/jballerina-unit-test/src/test/resources/test-src/klass/class-def-negative.bal

```diff
@@ -13,3 +13,26 @@ type C object {
     *A;
 };
 
+type ServiceObject service object {
+    resource function get name() returns string;
+};
+
+service class ServiceClass {
+    *ServiceObject;
+}
+
+type ClientObject client object {
+    resource function get name() returns string;
+};
+
+client class ClientClass {
+    *ClientObject;
+}
+
+type ClientObject2 client object {
+    *ClientObject;
+};
+
+client class ClientClass2 {
+    *ClientObject2;
+}
```

Comment:
We can also include tests where what's included is a class, rather than just an object typedesc.